Lotto/Lotto/LottoSeller.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  LottoSellter.swift
+//  Lotto
+//
+//  Created by nylah.j on 2022/05/10.
+//
+
+import Foundation
+
+struct LottoSeller {
+    enum Error: LocalizedError {
+        case nonEnoughMoney
```

Comment:
```suggestion
        case notEnoughMoney
```
non보다는 not이 좀 더 보편적이지 않나요...?

Reply:
[1062ced](https://github.com/next-step/swift-lotto/pull/24/commits/1062ced01a749bc58031b526d59ab364ea0a7880) 커밋에서 반영했습니다! 피드백 감사합니다!!~